src/components/Breadcrumbs/Breadcrumbs.md

```diff
@@ -0,0 +1,25 @@
+A navigation component that indicates the user's current location in the nav hierarchy.
+
+View the [Rivet documentation for Breadcrumbs](https://rivet.uits.iu.edu/components/navigation/breadcrumb/).
+
+### Default breadcrumb
+
+```jsx
+<Breadcrumbs>
+    <a href="#">Home</a>
+    <a href="#">Files</a>
+    my-file.txt
+</Breadcrumbs>
+```
+
+### Call out example
```

Comment:
Callout*

Reply:
I don't know that the call out -> callout changes are blocking. The Rivet docs themselves are inconsistent in how they refer to it. It's "Callout" in the header for that section, but "call out" and "call-out" (implying it's two words) in the actual example and code. I think we should hew towards the stylizing in the code and file an issue in Rivet to make the documentation consistent.

Reply:
I agree we should be consistent and that this is not a blocking issue. FWIW foundation uses `.callout` and `Callout` https://foundation.zurb.com/sites/docs/callout.html but I really don't think this will offend anyone if we name it `Call out`